requirements.txt

```diff
@@ -0,0 +1,2 @@
+fonttools>=4.7.0
+-e git://github.com/googlefonts/nototools.git#egg=master
```

Comment:
Yuck. @anthrotype , I'm thinking we push nototools to PIP and pull a version of it here.